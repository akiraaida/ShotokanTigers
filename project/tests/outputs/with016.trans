10 Octopus              00000 00000.00 S 
01 Octopus              12380 00005.00   
01 Octopus              12380 00000.10   
01 Octopus              12380 00005.00   
01 Octopus              12380 00000.10   
01 Octopus              12381 00005.00   
01 Octopus              12381 00000.05   
01 Octopus              12381 00005.00   
01 Octopus              12381 00000.05   
02 Octopus              12382 00005.00   
02 John Doe             12345 00005.00   
01 Octopus              12382 00000.10   
01 Octopus              12382 00005.00   
01 Octopus              12382 00000.10   
02 Octopus              12383 00005.00   
02 John Doe             12345 00005.00   
01 Octopus              12383 00000.05   
01 Octopus              12383 00005.00   
01 Octopus              12383 00000.05   
03 Octopus              12384 00005.00 TV
01 Octopus              12384 00000.10   
01 Octopus              12384 00005.00   
01 Octopus              12384 00000.10   
03 Octopus              12385 00005.00 TV
01 Octopus              12385 00000.05   
01 Octopus              12385 00005.00   
01 Octopus              12385 00000.05   
04 Octopus              12386 00005.00   
01 Octopus              12386 00000.10   
01 Octopus              12386 00005.00   
01 Octopus              12386 00000.10   
04 Octopus              12387 00005.00   
01 Octopus              12387 00000.05   
01 Octopus              12387 00005.00   
01 Octopus              12387 00000.05   
00                      00000 00000.00   
