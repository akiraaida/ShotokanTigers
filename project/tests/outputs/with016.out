Please enter your session type:
Please enter a login name:
Please enter a command user:
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12380):
5.10
Daily Withdrawals Remaining (12380):
495.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12380):
0.00
Daily Withdrawals Remaining (12380):
490.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12381):
5.05
Daily Withdrawals Remaining (12381):
495.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12381):
0.00
Daily Withdrawals Remaining (12381):
490.00
Please enter the account holder's name:
Please enter the account number:
Enter account number to transfer to:
Enter amount:
Success, 5.00 has been transferred to 12345.
Available Balance (12382):
5.10
Available Balance (12345):
115.00
Daily Transfer Limit Remaining (12382):
995.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12382):
0.00
Daily Withdrawals Remaining (12382):
495.00
Please enter the account holder's name:
Please enter the account number:
Enter account number to transfer to:
Enter amount:
Success, 5.00 has been transferred to 12345.
Available Balance (12383):
5.05
Available Balance (12345):
120.00
Daily Transfer Limit Remaining (12383):
995.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12383):
0.00
Daily Withdrawals Remaining (12383):
495.00
Please enter the account number:
Please enter recipient to be paid:
Please enter amount to pay:
Success, 5.00 has been paid to TV.
Available Balance (12384):
5.10
Daily Payment Limit to TV Remaining (12384):
1995.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12384):
0.00
Daily Withdrawals Remaining (12384):
495.00
Please enter the account number:
Please enter recipient to be paid:
Please enter amount to pay:
Success, 5.00 has been paid to TV.
Available Balance (12385):
5.05
Daily Payment Limit to TV Remaining (12385):
1995.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12385):
0.00
Daily Withdrawals Remaining (12385):
495.00
Please enter the account holder's name:
Please enter the account number:
Please enter amount to deposit:
Success, 5.00 has been deposited.
Available Balance (12386):
5.10
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12386):
0.00
Daily Withdrawals Remaining (12386):
495.00
Please enter the account holder's name:
Please enter the account number:
Please enter amount to deposit:
Success, 5.00 has been deposited.
Available Balance (12387):
5.05
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12387):
0.00
Daily Withdrawals Remaining (12387):
495.00
Please enter a command: 
